test/regression.uts

```diff
@@ -1639,7 +1639,8 @@ retry_test(_test)
 def _test():
     ret = conf.AS_resolver.resolve("8.8.8.8", "8.8.4.4")
     assert (len(ret) == 2)
-    all(x[1] == "AS15169" for x in ret)
```

Comment:
Not sure, but that's the AS currently reported by `riswhois.ripe.net`. I just added that value so that the tests pass.

Reply:
I looks like another temporary issue that we should not fix.

Reply:
Agreed it looks temporary. Question is: do we want our tests to fail when it happens? I suppose it depends on how often it happens and how long it lasts

Reply:
FTR, we still have the issue:
```
$ whois -h riswhois.ripe.net  2001:4860:4860::8888
% This is RIPE NCC's Routing Information Service
% whois gateway to collected BGP Routing Tables, version2.0
% IPv4 or IPv6 address to origin prefix match
%
% For more information visit http://www.ripe.net/ris/riswhois.html
%
% Connected to backend ris-whois14.ripe.net

route6:       2001:4860::/32
origin:       AS15169
descr:        GOOGLE, US
lastupd-frst: 2022-06-07 07:26Z  2001:7f8:30:0:1:1:0:8218@rrc05
lastupd-last: 2023-08-01 07:54Z  2001:7f8:20:101::210:52@rrc13
seen-at:      rrc00,rrc01,rrc03,rrc04,rrc05,rrc06,rrc07,rrc10,rrc11,rrc12,rrc13,rrc15,rrc16,rrc18,rrc19,rrc20,rrc21,rrc22,rrc23,rrc24,rrc25,rrc26
num-rispeers: 420
source:       RISWHOIS

route6:       2001:4860:4860::8888/128
origin:       AS4205777769
descr:         
lastupd-frst: 2023-07-31 14:35Z  2001:7f8:24::123@rrc20
lastupd-last: 2023-07-31 14:35Z  2001:7f8:24::123@rrc20
seen-at:      rrc20
num-rispeers: 1
source:       RISWHOIS
```

Reply:
It is safe to ignore this error. Last a similar issue happened was 2 to 3 years ago. We could print ret to ease debugging if it happens again.

The current output shows a local leak only seen by a single RRC instance.

Reply:
I don't like the idea of keeping a test that sometimes "does not work". Also, while we could ignore it, when this test fails, it can hide another test, running after this one, that would fail. For this reason I'm not confident in merging e.g. #4083.